src/main/kotlin/no/nav/syfo/db/Database.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.syfo.db
+
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import no.nav.syfo.DbEnv
+import org.flywaydb.core.Flyway
+import java.sql.Connection
+
+const val postgresJdbcPrefix = "jdbc:postgresql"
+
+interface DatabaseInterface {
+    val connection: Connection
+}
+
+class Database(val env: DbEnv) : DatabaseInterface {
+
+    val hikariDataSource = HikariDataSource(HikariConfig().apply {
+        jdbcUrl = generateJdbcUrlFromEnv(env)
+        username = env.dbUsername
+        password = env.dbPassword
+        maximumPoolSize = 2
+        minimumIdle = 1
+        isAutoCommit = false
+        transactionIsolation = "TRANSACTION_REPEATABLE_READ"
```

Comment:
Jeg ser at vi har av og til feil i prod som kan være relatert til den typen `transactionIsolation`:
[kibana](https://logs.adeo.no/app/discover#/view/4ce734f0-9646-11eb-a3f3-5be089f1b0b9?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-90d,to:now))&_a=(columns:!(message,envclass,level,application,host),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:envclass,negate:!f,params:(query:p),type:phrase),query:(match_phrase:(envclass:p))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:level,negate:!f,params:(query:Error),type:phrase),query:(match_phrase:(level:Error))),('$state':(store:appState),meta:(alias:eSYFO-applikasjoner,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:application,negate:!f,params:!(oppfolgingsplan,dialogmotearbeidsgiver,syfoapi,syfobrukertilgang,syfomotebehov,syfooppdfgen,syfooppfolgingsplanservice,syfooprest,esyfovarsel,dialogmote-frontend,esyfovarsel-job,esyfovarsel-planlegging-job,esyfo-info-frontend),type:phrases,value:'oppfolgingsplan,%20dialogmotearbeidsgiver,%20syfoapi,%20syfobrukertilgang,%20syfomotebehov,%20syfooppdfgen,%20syfooppfolgingsplanservice,%20syfooprest,%20esyfovarsel,%20dialogmote-frontend,%20esyfovarsel-job,%20esyfovarsel-planlegging-job,%20esyfo-info-frontend'),query:(bool:(minimum_should_match:1,should:!((match_phrase:(application:oppfolgingsplan)),(match_phrase:(application:dialogmotearbeidsgiver)),(match_phrase:(application:syfoapi)),(match_phrase:(application:syfobrukertilgang)),(match_phrase:(application:syfomotebehov)),(match_phrase:(application:syfooppdfgen)),(match_phrase:(application:syfooppfolgingsplanservice)),(match_phrase:(application:syfooprest)),(match_phrase:(application:esyfovarsel)),(match_phrase:(application:dialogmote-frontend)),(match_phrase:(application:esyfovarsel-job)),(match_phrase:(application:esyfovarsel-planlegging-job)),(match_phrase:(application:esyfo-info-frontend)))))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:message,negate:!f,params:(query:'Error%20in%20%5BMER_VEILEDNING_VARSEL%5D%20planner:%20%7C%20ERROR:%20could%20not%20serialize%20access%20due%20to%20concurrent%20update'),type:phrase),query:(match_phrase:(message:'Error%20in%20%5BMER_VEILEDNING_VARSEL%5D%20planner:%20%7C%20ERROR:%20could%20not%20serialize%20access%20due%20to%20concurrent%20update')))),index:'96e648c0-980a-11e9-830a-e17bbd64b4db',interval:auto,query:(language:lucene,query:''),sort:!()))

Reply:
Bær kanskje være `READ COMMITTED`

Reply:
Takk! 👍 Skal fikse